dll/cpl/input/input_list.c

```diff
@@ -323,14 +323,56 @@ InputList_AddInputMethodToUserRegistry(DWORD dwIndex, INPUT_LIST_NODE *pNode)
         }
     }
 
+    /* 
+     * First time more then 1 input language exists switching hotkeys needed
+     * set default values for language/layout switching hotkeys
+     */
+    if (2 == dwIndex) {
+        if (RegOpenKeyExW(HKEY_CURRENT_USER,
+                      L"Keyboard Layout\\Toggle",
```

Comment:
Please fix indentation (here and in the other functions below).

Reply:
done